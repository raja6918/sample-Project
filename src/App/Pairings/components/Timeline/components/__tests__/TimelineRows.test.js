import React from 'react';
import { mount } from 'enzyme';
import { TimelineRows } from '../';

const minProps = {
  rows: [{}, {}, {}, {}, {}, {}],
  rowsContainerRef: jest.fn(),
};

test.skip('TimelineRows Component renders', () => {
  expect(mount(<TimelineRows {...minProps} />)).toMatchSnapshot();
});
