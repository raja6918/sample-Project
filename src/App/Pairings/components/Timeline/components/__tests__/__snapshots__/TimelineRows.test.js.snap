// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineRows Component renders 1`] = `
<TimelineRows
  rows={
    Array [
      Object {},
      Object {},
      Object {},
      Object {},
      Object {},
      Object {},
    ]
  }
  rowsContainerRef={
    [MockFunction] {
      "calls": Array [
        Array [
          <div
            class="pt-horizontal-lines"
          >
            <div
              class="pt-hl"
              style="height: 40px;"
            />
            <div
              class="pt-hl"
              style="height: 40px;"
            />
            <div
              class="pt-hl"
              style="height: 40px;"
            />
            <div
              class="pt-hl"
              style="height: 40px;"
            />
            <div
              class="pt-hl"
              style="height: 40px;"
            />
            <div
              class="pt-hl"
              style="height: 40px;"
            />
          </div>,
        ],
      ],
    }
  }
>
  <div
    className="pt-horizontal-lines"
  >
    <div
      className="pt-hl"
      key="0"
      style={
        Object {
          "height": 40,
        }
      }
    />
    <div
      className="pt-hl"
      key="1"
      style={
        Object {
          "height": 40,
        }
      }
    />
    <div
      className="pt-hl"
      key="2"
      style={
        Object {
          "height": 40,
        }
      }
    />
    <div
      className="pt-hl"
      key="3"
      style={
        Object {
          "height": 40,
        }
      }
    />
    <div
      className="pt-hl"
      key="4"
      style={
        Object {
          "height": 40,
        }
      }
    />
    <div
      className="pt-hl"
      key="5"
      style={
        Object {
          "height": 40,
        }
      }
    />
  </div>
</TimelineRows>
`;
